RUNBOOK.md

```diff
@@ -0,0 +1,153 @@
+
+Secrets with information from [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments) will be managed via the terraform code in there.
+
+The contents of the `manage-recalls-ui` secret are handled in the following way:
+
+- `API_CLIENT_ID` - managed externally via tooling in the `hmpps-auth` project.
+- `API_CLIENT_SECRET` - managed externally via tooling in the `hmpps-auth` project.
+- `APPINSIGHTS_INSTRUMENTATIONKEY` - managed externally via the [dps-project-bootstrap](https://github.com/ministryofjustice/dps-project-bootstrap) tooling.
+- `MANAGER_NAME` - set manually by the [#ppud-replacement-devs](https://mojdt.slack.com/archives/C0223AGGQU8) team.
+- `MANAGER_PHONE` - set manually by the [#ppud-replacement-devs](https://mojdt.slack.com/archives/C0223AGGQU8) team.
+- `OS_PLACES_API_KEY` - set manually by the [#ppud-replacement-devs](https://mojdt.slack.com/archives/C0223AGGQU8) team.
+- `SESSION_SECRET` - managed externally via the [dps-project-bootstrap](https://github.com/ministryofjustice/dps-project-bootstrap) tooling.
+- `SYSTEM_CLIENT_ID` - legacy secret no longer used (and set blank) but still needed for the application to start up (due to code coming from the template repo).
+- `SYSTEM_CLIENT_SECRET` - legacy secret no longer used (and set blank) but still needed for the application to start up (due to code coming from the template repo).
+
+## System backup and restore
+
+This is handled by the HMPPS Cloud Platform Team, but details of how each component is considered is below.
+
+### Kubernetes Resources
+
+Nightly backups of cluster objects/resources are taken using a tool called [velero](https://velero.io/) - this will allow efficient recovery from backup should this need to be done.
+
+### Database/Object Persistence
+
+Redis (Elasticache) is used as a performance cache, there is no need for backups to be taken as the data held within is temporary and can be rebuilt automatically.
+
+## Monitoring and alerting
+
+### Log aggregation solution
+
+Please see [Confluence](https://dsdmoj.atlassian.net/wiki/spaces/PUD/pages/3622830168/Monitoring+Operability#Logging) for more details.
+
+### Log message format
+
+Logs are output in structured JSON. This isn't very pretty in Elasticsearch/Kibana, but is processed correctly in Application Insights.
+
+### Events and error messages
+
+Please see [Confluence](https://dsdmoj.atlassian.net/wiki/spaces/PUD/pages/3622830168/Monitoring+Operability#Runtime-Error-Reporting) for more details.
+
+### Metrics
+
+Please see [Confluence](https://dsdmoj.atlassian.net/wiki/spaces/PUD/pages/3622830168/Monitoring+Operability#Metrics) for more details.
+
+### Health checks
+
+#### Health of dependencies
+
+`/health` (i.e. https://manage-recalls.hmpps.service.justice.gov.uk/health) checks and reports the health of all services and components that the application depends upon. A HTTP 200 response code indicates that everything is healthy.
```

Comment:
Is it worth linking to where the list of dependent services is listed in the code? 
I need to add postcode API to that list btw

Reply:
If you can point me to where it is I can link to it 😄